kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/integration/BaseConnectorIT.java

```diff
@@ -149,16 +155,17 @@ protected void clearPriorTable(BigQuery bigQuery, String table) {
   }
 
   protected void waitForCommittedRecords(
-      String connector, String topic, long numRecords, int numTasks
+      String connector, long numRecords, int numTasks, String topics
```

Comment:
What was the idea behind this change?

Reply:
Ah, good catch. I prepped this PR a while ago and some things got shuffled around between then and now; this change is unnecessary and I've reverted it in the latest push.